Sources/SwiftParser/generated/ExperimentalFeatures.swift

```diff
@@ -44,4 +44,27 @@ extension Parser.ExperimentalFeatures {
 
   /// Whether to enable the parsing of ValueGenerics.
   public static let valueGenerics = Self (rawValue: 1 << 6)
+
+  /// Creates a new value representing the experimental feature with the
+  /// given name, or returns nil if the name is not recognized.
+  public init?(name: String) {
+    switch name {
+    case "referenceBindings":
```

Comment:
I think these feature names should match the feature names in the compiler (ie. experimental or upcoming features), which means that they will need to be manually specified.